classes/query/QueryDAO.php

```diff
@@ -276,29 +277,31 @@ public function deleteObject($query)
      */
     public function deleteById($queryId, $assocType = null, $assocId = null)
     {
-        $params = [(int) $queryId];
+        $queryQueries = DB::table('queries')
```

Comment:
Unfortunately it should be called `$queryQuery` :clown_face:

Reply:
:) Changed